cdap-docs/reference-manual/source/http-restful-api/http-restful-api-v3/metrics.rst

```diff
@@ -98,114 +373,162 @@ The general form of a metrics request is::
+
+  {"startTime":0,"endTime":0,"series":[{"metricName":"system.process.events.processed",
+   "grouping":{},"data":[{"time":0,"value":11188}]}]}
 
 If the run-ID is not specified, we aggregate the events processed for all the runs of this flow.
 
 If you want the number of input objects processed across all Flowlets of a Flow, you address the metrics
 API at the Flow context::
 
-  GET <base-url>/metrics/system/apps/CountRandom/flows/
-    CountRandomFlow/process.events.processed?start=now-5s&count=5
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom.flow.CountRandom.flowlet.*
+    &metric=system.process.events.processed&start=now-5s&count=5
+
+Similarly, you can address the context of all Flows of an Application, an entire Application, or the entire 
+namespace of a CDAP instance::
+
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom.flow.*
+    &metric=system.process.events.processed&start=now-5s&count=5
 
-Similarly, you can address the context of all flows of an Application, an entire Application, or the entire CDAP::
+  POST <base-url>/metrics/query?context=namespace.default.app.CountRandom
+    &metric=system.process.events.processed&start=now-5s&count=5
 
-  GET <base-url>/metrics/system/apps/CountRandom/
-    flows/process.events.processed?start=now-5s&count=5
-  GET <base-url>/metrics/system/apps/CountRandom/
-    process.events.processed?start=now-5s&count=5
-  GET <base-url>/metrics/system/process.events?start=now-5s&count=5
+  POST <base-url>/metrics/query?context=namespace.default
+    &metric=system.process.events.processed&start=now-5s&count=5
 
 To request user-defined metrics instead of system metrics, specify ``user`` instead of ``cdap`` in the URL
 and specify the user-defined metric at the end of the request.
 
 For example, to request a user-defined metric for the *HelloWorld* Application's *WhoFlow* Flow::
 
-  GET <base-url>/metrics/user/apps/HelloWorld/flows/
-    WhoFlow/flowlets/saver/names.bytes?aggregate=true
+  POST <base-url>/metrics/query?context=namespace.default.app.HelloWorld.flow.WhoFlow.flowlet.saver
+    &metric=user.names.bytes&aggregate=true
 
-To retrieve multiple metrics at once, instead of a GET, issue an HTTP POST, with a JSON list as the request body that enumerates the name and attributes for each metrics. For example::
+Retrieving multiple metrics at once, by issuing an HTTP POST request with a JSON list as
+the request body that enumerates the name and attributes for each metric, is currently not
+supported in this API. Instead, use the :ref:`v2 API
+<http-restful-api-v2-metrics-multiple>`. It will be supported in a future release.
 
-  POST <base-url>/metrics
 
-with the arguments as a JSON string in the body::
+.. _http-restful-api-metrics-tag-list:
 
-  Content-Type: application/json
-  [ "/system/collect.events?aggregate=true",
-  "/system/apps/HelloWorld/process.events.processed?start=1380323712&count=6000" ]
+Tag List
```

Comment:
why it has tag list as title? Maybe call it Querying for multiple time-series using groupBy? Or Querying for break down by tag values using groupBy? Or Using groupBy to break down result timeseries? Or Using groupBy? Or ...